app/views/pages/hiring_content.html.haml

```diff
@@ -0,0 +1,145 @@
+        This is a hands-on role, where you'll be writing and editor content, alongside planning and leading our volunteer maintainers.
+        Currently this role is split between the CEO and Head of Open Source, both of whom you'll work closely with.
+      %p.text-p-xlarge.mb-32
+        We are building an exceptional team of ambitious, passionate people who love their work, excel at what they do, and have fun doing it.
+        The values of empathy, integrity and personal-growth are core to everything we do.
+        If that sounds exciting to be part of, and you feel you have the skills to help, we'd love for you to apply.
+
+      %h2.text-h2.mb-12 Responsibilities
+      %p.text-p-xlarge.mb-16
+        Your role is to fundmentally take all the work currently done by the CEO and the Head of Open Source,
+        Your role splits roughly into three areas.
+      %p.text-p-xlarge.mb-12
+        The first is strategic, which has the following key responsibilities:
+      %ul.text-p-xlarge.mb-16
+        %li Work with our CEO, Head of Open Source to more tighly define our our wider content strategy for the next three years.
+        %li Work with our senior maintainers to design and document systems that improve the flow and quality of content.
+        %li Work with our content partners (e.g. Packt Publishing) to determine synergies between their content and ours, and how we can make the most use of their educational portfolio.
+      %p.text-p-xlarge.mb-12
+        The second is a leadership role:
+      %ul.text-p-xlarge.mb-16
+        %li You will be the senior content voice for our community, answering questions, arbitrating discussions, and acting as the knowledgable sage of wisdom. This doesn't mean you have to be the most knowledgable on any given subject, but means you must speak to Exercism's perspective on content, education, and how we make pragmatic decisions.
+        %li You will act as a locum maintainer - supporting tracks where there are no maintainers, addressing issues and reviewing pull-requests from contributors. In this you will be supported by our community "reviewers" team, who help out in PR reviews on tracks where there are no formal maintainers.
+      %p.text-p-xlarge.mb-16
+        The third, but by no means least, is actually writing educational content.
+        As part of our three year plan, we are extending our content beyond programming-language content, to the programming concept - looking at fundemental programming concepts, design patters, and the knowledge that makes someone an excellent programmer.
+        You will lead this, planning out and writing the content, supported by both our internal team and our contributor team.
+
+      %p.text-p-xlarge.mb-40
+        Although that's a lot of different hats to wear, they are a synegistic set of responsibilities, which all center around the central mindset of "owning" Exercism's content.
```

Comment:
> synegistic

synergistic

> central mindset of "owning" Exercism's content.

I think we should specify that this is the content that has to do with programming knowledge. For example, they wouldn't necessarily own marketing content and, say, hiring pages.